domains.txt

```diff
@@ -617,27 +685,34 @@ sjit.co.uk
 skala.co.uk
 skillrecruit.com
 skillsearch.com
+skypath.co.uk
 smart-co.co.uk
 smartedgesolutions.co.uk
 smartsourcing.plc.uk
 snap.hr
+socode.co.uk
 softwarepersonnel.co.uk
 solagroup.com
 soprasteria.com
+soultek.co.uk
 sourced.ai
 sourced.tech
 southern-lights.co.uk
 spectrumit.co.uk
 speerhead.com
 spencer-rose.com
 spgsearch.com
+spherelondon.co.uk
 spring.com
+sqasearch.co.uk
 sqcp.com
 squareone.eu
 squareoneresources.com
 squareoneresourcesnews.com
+staffhost.co.uk
```

Comment:
Today, StaffHost is a specialist recruiter that works with leading blue-chips through to dynamic SME’s within Global Technology markets. We place permanent and contract professionals globally across Technology & Cyber Security. We have a demonstrable track record in delivering